exercises/practice/bottle-song/.meta/example.ex

```diff
@@ -0,0 +1,51 @@
+defmodule BottleSong do
+  @moduledoc """
+  Handles lyrics of the popular children song: Ten Green Bottles
+  """
+
+  @quantifiers %{
+    10 => "ten",
+    9 => "nine",
+    8 => "eight",
+    7 => "seven",
+    6 => "six",
+    5 => "five",
+    4 => "four",
+    3 => "three",
+    2 => "two",
+    1 => "one",
+    0 => "no"
+  }
+
+  @spec recite(pos_integer, pos_integer) :: Enum.t()
```

Comment:
```suggestion
  @spec recite(pos_integer, pos_integer) :: String.t()
```
same for the stub file